src/coreclr/inc/readytorun.h

```diff
@@ -408,9 +408,7 @@ enum ReadyToRunHelper
     READYTORUN_HELPER_DblRem                    = 0xE0,
     READYTORUN_HELPER_FltRem                    = 0xE1,
 
-    // These two helpers can be removed once MINIMUM_READYTORUN_MAJOR_VERSION is 10+
-    // alongside the CORINFO_HELP_FLTROUND/CORINFO_HELP_DBLROUND
-    // counterparts and all related code.
+    // Unused since READYTORUN_MAJOR_VERSION 10.0
     READYTORUN_HELPER_DblRound                  = 0xE2,
```

Comment:
does it mean we can only remove them when we bump to 11 now?

Reply:
We have been keeping the old no longer used R2R constants around to make R2R dump work for old images.

Reply:
For example: https://github.com/dotnet/runtime/blob/040fde48a75f0c211353f073e4f69e2e31607752/src/coreclr/inc/readytorun.h#L91